lib/perl/Genome/Config.pm

```diff
@@ -130,6 +130,12 @@ sub all_keys {
 
 sub config_subpath { Path::Class::File->new('genome', 'config.yaml') }
 
+sub project_dir {
+    my $dir = $ENV{XGENOME_CONFIG_PROJECT_DIR};
+    return $dir if $dir;
```

Comment:
It may not matter, but it appears that `home_dir` wraps its path in a `Path::Class::Dir` object on return. It seems odd that `project_dir` and `home_dir` would have inconsistent return types.